### DISCUSSION TITLE
Failed to install io.github.ungoogled_software.ungoogled_chromium: Error pulling from repo: syncfs: Input/output error

### DISCUSSION THREAD

**Discussion by Sinaaaa** [Eloston/ungoogled-chromium#3030] [Q&A]
I'm using the Ungoogled flatpak. For the past couple of days I could not update / install Ungoogled on my 2 machines.

```
Error: Error pulling from repo: syncfs: Input/output error
error: Failed to install io.github.ungoogled_software.ungoogled_chromium: Error pulling from repo: syncfs: Input/output error

```

Anyone has an idea how to fix this? (I can provide more information later, if this is more system specific)

**Comment by rany2** [comment]
Make an issue on flatpak's tracker. This is a general issue with flatpak and not related to UGC.
